lib/v2/screens/explore_screens/vote_screens/proposals/usecases/get_proposals_use_case.dart

```diff
@@ -3,7 +3,15 @@ import 'package:seeds/v2/datasource/remote/api/proposals_repository.dart';
 import 'package:seeds/v2/screens/explore_screens/vote_screens/vote/interactor/viewmodels/proposal_type_model.dart';
 
 class GetProposalsUseCase {
-  Future<Result> run(ProposalType proposalType) {
-    return ProposalsRepository().getProposals(proposalType);
+  final ProposalsRepository _proposalsRepository = ProposalsRepository();
+
+  Future<List<Result>> run(ProposalType proposalType) {
+    var futures = [
+      _proposalsRepository.getProposals(proposalType),
+      _proposalsRepository.getSupportLevels('alliance'),
+      _proposalsRepository.getSupportLevels('campaign'),
+      _proposalsRepository.getSupportLevels('milestone'),
+    ];
+    return Future.wait(futures);
   }
```

Comment:
@n13 Could you confirm if this is what you want?
1 - I bring the voice_need for each type of campaign.

Here I do not understand why a single collection is not used that contains all 3 support leves, that way I would only make one call.

Reply:
It's because it's blockchain and that's the way we structured it - not ideal for retrieval in this case. 

The EOSIO data structures are primitive. 

There may have been a way to make this one call, not sure, but we didn't do that.